OpenSim/Simulation/Model/PhysicalFrame.cpp

```diff
@@ -195,11 +195,36 @@ void PhysicalFrame::updateFromXMLNode(SimTK::Xml::Element& aNode, int versionNum
                     SimTK::Xml::element_iterator geomSetIter = visObjElement.element_begin("GeometrySet");
                     if (geomSetIter != visObjElement.element_end()) {
                         convertDisplayGeometryToGeometryXML(aNode, outerScaleFactors, outerTransform, *geomSetIter);
-                        geomSetIter->setElementTag("geometry");
                     }
                 }
             }
         }
```

Comment:
Just double checking that we agreed that this should be moved to `ModelComponent` since `<geometry>` was housed there? The test passes because the Model only had `Geometry` on `PhysicalFrames`, which was just luck.
Otherwise, we can revert the test file to an earlier format (30000) then this issue goes away since `<geometry>` on `ModelComponent` did not exist then. Something to resolve after dev week.